contracts/core/extensions/CoreIssuance.sol

```diff
@@ -56,6 +56,31 @@ contract CoreIssuance is
     {
         issueInternal(
             msg.sender,
+            msg.sender,
+            _set,
+            _quantity
+        );
+    }
+
+    /**
+     * Exchange components for Set tokens
```

Comment:
Can we clarify here: 1. who the components come from (caller, param, or vault), 2. who gets the Set (caller, param, or vault)

Reply:
+1